### ISSUE TITLE
Windows cross compilation fail

### ISSUE THREAD

**Issue by andyoknen** [pocketnetteam/pocketnet.core#259]
```
  CXXLD    pocketcoind.exe
libpocketcoin_server.a(libpocketcoin_server_a-init.o):init.cpp:(.text$_ZN9SimpleWeb12SocketServerIN5boost4asio19basic_stream_socketINS2_2ip3tcpENS2_8executorEEEE6acceptEv[_ZN9SimpleWeb12SocketServerIN5boost4asio19basic_stream_socketINS2_2ip3tcpENS2_8executorEEEE6acceptEv]+0x80f): undefined reference to `AcceptEx'
libpocketcoin_server.a(libpocketcoin_server_a-init.o):init.cpp:(.text$_ZN5boost4asio6detail25win_iocp_socket_accept_opINS0_12basic_socketINS0_2ip3tcpENS0_8executorEEES5_ZN9SimpleWeb12SocketServerINS0_19basic_stream_socketIS5_S6_EEE6acceptEvEUlRKNS_6system10error_codeEE_NS1_18io_object_executorIS6_EEE11do_completeEPvPNS1_18win_iocp_operationESG_y[_ZN5boost4asio6detail25win_iocp_socket_accept_opINS0_12basic_socketINS0_2ip3tcpENS0_8executorEEES5_ZN9SimpleWeb12SocketServerINS0_19basic_stream_socketIS5_S6_EEE6acceptEvEUlRKNS_6system10error_codeEE_NS1_18io_object_executorIS6_EEE11do_completeEPvPNS1_18win_iocp_operationESG_y]+0x181): undefined reference to `GetAcceptExSockaddrs'
libpocketcoin_server.a(libpocketcoin_server_a-init.o):init.cpp:(.text$_ZN5boost4asio6detail25win_iocp_socket_accept_opINS0_12basic_socketINS0_2ip3tcpENS0_8executorEEES5_ZN9SimpleWeb12SocketServerINS0_19basic_stream_socketIS5_S6_EEE6acceptEvEUlRKNS_6system10error_codeEE_NS1_18io_object_executorIS6_EEE11do_completeEPvPNS1_18win_iocp_operationESG_y[_ZN5boost4asio6detail25win_iocp_socket_accept_opINS0_12basic_socketINS0_2ip3tcpENS0_8executorEEES5_ZN9SimpleWeb12SocketServerINS0_19basic_stream_socketIS5_S6_EEE6acceptEvEUlRKNS_6system10error_codeEE_NS1_18io_object_executorIS6_EEE11do_completeEPvPNS1_18win_iocp_operationESG_y]+0x538): undefined reference to `AcceptEx'
collect2: error: ld returned 1 exit status
Makefile:8463: recipe for target 'pocketcoind.exe' failed
make[2]: *** [pocketcoind.exe] Error 1
make[2]: Leaving directory '/home/build/win/src/src'
Makefile:20001: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/home/build/win/src/src'
Makefile:805: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1
```

**Comment by lostystyg** [comment]
Should be fixed in 0c5c9ada506024ff00659f6d0b1cfdfb1e6a8dfe

**Comment by lostystyg** [comment]
`make deploy` also failed because of missing *pocketcoin-wallet* binary that was temporarily disabled because of lots building issues. 5e036830024cd5f69edb5988d13c86399fbeef98 fully disables *pocketcoin-wallet* and fixes `make deploy`.
